packages/amplify-codegen-e2e-tests/test-apps/graphql-generator-app/cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
All the lines on this file are comments. Should we remove this file?

Reply:
This file was auto-generated by cypress. It needs to be present for the cypress to function. I think we could remove all of the lines though.